api/users/employment-schema.json

```diff
@@ -0,0 +1,52 @@
+{
+	"type": "array",
+	"items": {
+		"type": "object",
+		"properties": {
+			"title": {
+				"type": "string"
+			},
+			"startYear": {
+				"type": "integer",
+				"minimum": 1900
+			},
+			"startMonth": {
+				"type": "integer",
+				"minimum": 1,
+				"maximum": 12
+			},
+			"endMonth": {
+				"type": "integer",
+				"minimum": 1,
+				"maximum": 12
+			},
+			"endYear": {
+				"type": "integer",
+				"minimum": 1900
+			},
+			"ongoing": {
+				"type": "boolean"
+			},
+			"department": {
+				"type": "string"
+			},
+			"institution": {
+				"type": "string",
+				"minLength": 1
+			}
+		},
+		"required": [
+			"startYear",
+			"startMonth",
+			"ongoing",
+			"department",
+			"institution",
+			"title"
+		],
+		"additionalProperties": false,
+		"dependencies": {
```

Comment:
No, they are not necessary because `startMonth` and `startYear` are both always required no matter what, so they're not really dependencies of each other.